integration/kubernetes/k8s-limit-range.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	get_pod_config_dir
+	namespace_name="default-cpu-example"
+	pod_name="default-cpu-test"
+}
+
+@test "Limit range for storage" {
+	wait_time=20
+	sleep_time=2
+
+	# Create namespace
+	kubectl create namespace "$namespace_name"
+
+	# Create the LimitRange in the namespace
+	kubectl create -f "${pod_config_dir}/limit-range.yaml" --namespace=${namespace_name}
+
+	# Create the pod
+	kubectl create -f "${pod_config_dir}/pod-cpu-defaults.yaml" --namespace=${namespace_name}
+
+	# Get pod specification
+	cmd="kubectl get pod "$pod_name" -o jsonpath='{.status.phase}' --namespace="$namespace_name" | grep Running"
```

Comment:
Is this only checking that the pod is running? 
If so, we could use `kubectl wait`. 
Also, if this test is about Limits, I think that we should also check those limits were properly set,  right?

Reply:
@chavafg , I tried it with `kubectl wait` but it does not retrieve the information so maybe it does not work with a namespace

Reply:
did you tried with `-n $namespace_name`? 
Something like: 
```
kubectl wait --for condition=Ready pod "$pod_name" -n "$namespace_name"
```

Reply:
@chavafg , I fixed it with `wait` , I needed to add a command at the yaml so the container stays running